crates/router/src/services/authorization/roles.rs

```diff
@@ -78,6 +84,38 @@ impl RoleInfo {
             })
     }
 
+    #[cfg(feature = "recon")]
+    pub fn get_recon_acl(&self) -> HashMap<Resource, ReconPermissionScope> {
```

Comment:
Can we move this function as well to the recon files. It's not role's responsibility of the role to give the recon acl, but it's other way around.

Reply:
Instead of making a dangling fn, we can let it be inside the `RoleInfo` impl, since it relies only on the RoleInfo struct